src/main/java/cc/polyfrost/oneconfig/hud/Hud.java

```diff
@@ -208,4 +221,9 @@ public void setScale(float scale, boolean example) {
             name = "Show in GUIs"
     )
     public boolean showInGuis = true;
+
+    @Switch(
+            name = "Show in SkyBlock"
+    )
+    public boolean showInSkyBlock = true;
```

Comment:
I feel like this shouldn't be here for all HUD elements, it just doesn't make sense for:
1. mods like SkyGuide which well are only active in skyblock
2. mods not targeted at Hypixel which might not even be active on Hypixel

I feel like a better solution would be to postpone this to when profiles are fully added, and then add something like automatic profile switching based on gamemode.
If you insist on adding this now I would change it to "Show in Hypixel Skyblock" to avoid confusion with non-Hypixel players (especially for modern mc versions later down the line)